Sources/Subs-Calendar.php

```diff
@@ -699,15 +697,25 @@ function getCalendarList($start_date, $end_date, $calendarOptions)
 	// Give birthdays and holidays a friendly format, without the year
 	$date_format = str_replace(array('%Y', '%y', '%G', '%g', '%C', '%c', '%D'), array('', '', '', '', '', '%b %d', '%m/%d'), get_date_or_time_format('date'));
 
+	// timeformat() uses $user_info to get the format
+	if (isset($user_info['time_format']))
+		$saved_dt_format = $user_info['time_format'];
+	$user_info['time_format'] = $date_format;
 	foreach (array('birthdays', 'holidays') as $type)
 	{
 		foreach ($calendarGrid[$type] as $date => $date_content)
 		{
-			$date_local = preg_replace('~(?<=\s)0+(\d)~', '$1', trim(timeformat(strtotime($date), $date_format), " \t\n\r\0\x0B,./;:<>()[]{}\\|-_=+"));
+			// Make sure to apply no offsets
+			$date_local = preg_replace('~(?<=\s)0+(\d)~', '$1', trim(timeformat(strtotime($date), true, true), " \t\n\r\0\x0B,./;:<>()[]{}\\|-_=+"));
```

Comment:
```suggestion
			// Make sure to apply no offsets
			$date_local = preg_replace('~(?<=\s)0+(\d)~', '$1', trim(timeformat(strtotime($date), $date_format, true), " \t\n\r\0\x0B,./;:<>()[]{}\\|-_=+"));
```
All that is needed to fix this issue is to set the `$offset_type` parameter to a value other than `false` or `'forum'`. 

Passing a string to the `$show_today` parameter is supported by the `timeformat()` function, although the function's documentation currently fails to explain this. It is probably a good idea to update the documentation to reflect this, since this behaviour is used in several places throughout the codebase where output in a particular format is required. (To be clear, I think this is counter-intuitive behaviour, but it's been like this since at least SMF 2.0, so we just have to live with it.)

Reply:
Thanks, good input.  I wasn't aware the string could be specified in that field.  

I made that clear in the comments & in the doc block.